src/Spice86.Core/Emulator/Devices/ExternalInput/Pic.cs

```diff
@@ -21,176 +23,288 @@
 /// <li>https://k.lse.epita.fr/internals/8259a_controller.html</li>
 /// </ul>
 /// </summary>
-public class Pic : DefaultIOPortHandler {
```

Comment:
Dune exhibits the following behavior :
- music stops
- FMVs stop at the first frame
- Voice lines stops and then the game stops responding because no IRQs is being serviced at all around the 26 million instruction mark.
- Sometimes it doesn't exit the intro in order to start a new game after the Esc key is used.